app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+    def page_title
+        @pg_title ||= "Haxxor News"
+    end
```

Comment:
Looks like you're using 4 spaces for indentation. The [general Ruby style](https://github.com/rubocop-hq/ruby-style-guide) is to use 2 spaces. I'd recommend using something like rubocop or an editor plugin to help you learn what those styles are and enforce them in your code. At the very least, being consistent is more important than being a "4 spaces" or "2 spaces" person.

Normally I wouldn't care about style in code reviews, but some folks really care when reviewing your github history.